persistence/src/test/java/cz/fi/muni/pa165/dao/UserDaoTest.java

```diff
@@ -48,7 +96,61 @@ public void findByGendre() {
         assertThat(men.size() == 1);

```

Comment:
toto nebude fungovat...  je treba prepsat na
`assertThat(men.size).isOne();`
(koukam ze jsem to nekde taky pouzil spatne)

Reply:
Ten test jsem nepsala ja